### PR TITLE
Card discussion component

### PR DESCRIPTION
**Pull request by GOWTHAM2K1**
### Contributor checklist

- [x] This pull request is on a [separate branch](https://docs.github.com/en/get-started/quickstart/github-flow) and not the main branch

---

### Description

Completed card discussion input component

- #ISSUE_NUMBER
- #460

### CODE REVIEW

**Comment by andrewtavis** [comment]
Don't worry about the backend check, @GOWTHAM2K1 :) Likely something on our end. I'll check this in the coming days and then we can make more issues for the frontend for the functionality 😊

**Comment by andrewtavis** [comment]
Thanks for this!

**Comment by GOWTHAM2K1** [comment]
> Don't worry about the backend check, @GOWTHAM2K1 :) Likely something on our end. I'll check this in the coming days and then we can make more issues for the frontend for the functionality 😊

thank you, Do let me know if you need anything to be done.

**Comment by to-sta** [comment]
@andrewtavis the `pr_ci_backend` check fails due to an addition of code from PR #421. That should have not been merged.

```python
DATABASES = {
    "default": {
        "ENGINE": "django.db.backends.postgresql",
        "NAME": DATABASE_NAME,
        "USER": DATABASE_USER,
        "PASSWORD": DATABASE_PASSWORD,
        "HOST": DATABASE_HOST,
        "PORT": DATABASE_PORT,
    },
    # This is not needed.
    "test": {
        "ENGINE": "django.db.backends.postgresql",
        "NAME": "test_" + DATABASE_NAME,
        "USER": DATABASE_USER,
        "PASSWORD": DATABASE_PASSWORD,
        "HOST": DATABASE_HOST,
        "PORT": DATABASE_PORT,
    },
}
```

We can delete the "test" dictionary complete.

```python
DATABASES = {
    "default": {
        "ENGINE": "django.db.backends.postgresql",
        "NAME": DATABASE_NAME,
        "USER": DATABASE_USER,
        "PASSWORD": DATABASE_PASSWORD,
        "HOST": DATABASE_HOST,
        "PORT": DATABASE_PORT,
    }
}
```
That would fix the issue 😄

**Comment by andrewtavis** [comment]
Thank you @to-sta! Good point on that :) Got rid of the files but left the references to them in another one :) I'll fix all this tomorrow 😴

File: frontend/components/card/search-result/CardSearchResult.vue [hunk 1]
```diff
@@ -27,6 +27,12 @@
     >
       <CardSearchResultUser :user="user" :isPrivate="isPrivate" />
     </div>
+    <div
```
Comment (andrewtavis) [inline frontend/components/card/search-result/CardSearchResult.vue:30]: I'm going to revert the changes to search result as this should be a stand alone component rather than an option of search results.

File: frontend/pages/home/index.vue [hunk 2]
```diff
@@ -40,6 +40,13 @@
         :isPrivate="false"
         :user="user"
       />
+
+      <CardSearchResult
```
Comment (andrewtavis) [inline frontend/pages/home/index.vue:44]: Will change this to referencing the component itself :)

**Review by andrewtavis — APPROVED** [review]
Generally all's looking good, @GOWTHAM2K1 :) Thanks for the work here. If you'd like to pick up another issue at some point, I'd suggest that you read through the [contributing guide](https://github.com/activist-org/activist/blob/main/CONTRIBUTING.md) we've made so that future PRs are better formatted and are following the [style guide](https://github.com/activist-org/activist/blob/main/STYLEGUIDE.md) 😊

Appreciate you jumping in and getting so much done!
